dotnet/src/Connectors/Connectors.Ollama.UnitTests/OllamaPromptExecutionSettingsTests.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Linq;
+using System.Text.Json;
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.Connectors.Ollama;
+using Xunit;
+
+namespace SemanticKernel.Connectors.Ollama.UnitTests;
+
+/// <summary>
+/// Unit tests of <see cref="OllamaPromptExecutionSettings"/>.
+/// </summary>
+public class OllamaPromptExecutionSettingsTests
+{
+    [Fact]
+    public void FromExecutionSettingsWhenAlreadyOllamaShouldReturnSameAsync()
+    {
+        // Arrange
+        var executionSettings = new OllamaPromptExecutionSettings();
+
+        // Act
+        var ollamaExecutionSettings = OllamaPromptExecutionSettings.FromExecutionSettings(executionSettings);
+
+        // Assert
+        Assert.Same(executionSettings, ollamaExecutionSettings);
+    }
+
+    [Fact]
+    public void FromExecutionSettingsWhenNullShouldReturnDefaultAsync()
+    {
+        // Arrange
+        OllamaPromptExecutionSettings? executionSettings = null;
+
+        // Act
+        var ollamaExecutionSettings = OllamaPromptExecutionSettings.FromExecutionSettings(executionSettings);
+
+        // Assert
+        Assert.Null(ollamaExecutionSettings.Stop);
+        Assert.Null(ollamaExecutionSettings.Temperature);
+        Assert.Null(ollamaExecutionSettings.TopP);
+        Assert.Null(ollamaExecutionSettings.TopK);
```

Comment:
There should also be a test to confirm that these values are not sent in the request content i.e. we are not sending default values.